models/istio_config.go

```diff
@@ -171,6 +174,11 @@ var IstioConfigHelpMessages = map[string][]IstioConfigHelp{
 	"k8shttproutes": { // TODO
 		{ObjectField: "", Message: "Kubernetes Gateway API Configuration Object. HTTPRoute is for multiplexing HTTP or terminated HTTPS connections."},
 	},
+	"k8sreferencegrants": {
+		{ObjectField: "", Message: "Kubernetes Gateway API Configuration Object. ReferenceGrant is for enabling cross namespace references within Gateway API."},
+		{ObjectField: "spec.from", Message: "Define the group, kind, and namespace of resources that may reference items described in the to list."},
+		{ObjectField: "spec.to", Message: "Define the group and kind of resources that may be referenced by items described in the from list."},
```

Comment:
How do I get this text to show up in the UI?

Reply:
now it should work, just clicking on the help icon in line.